client/src/components/resources/resources.css

```diff
@@ -0,0 +1,206 @@
+	background-color: #ff9617;
+	color: #212121;
+	transition: all 0.3s ease-in-out;
+	box-shadow: 0px 3px 0px 0px rgb(255, 150, 23);
+}
+
+.resources-tabs button:hover {
+	transition: all 0.3s ease-in-out;
+	box-shadow: 0px 2px 0px 0px rgb(255, 150, 23, 0.8);
+}
+
+/* Tab content */
+.resource-contents {
+	flex-grow: 1;
+	overflow: auto;
+	z-index: 2;
+}
+
+.resources-content {
+	color: white;
+	padding: 5% 5% 10% 5%;
+	display: none;
+	justify-content: center;
+	position: relative;
+	animation-name: arrivedContent;
+	animation-duration: 0.8s;
+}
+
+.resources-content.active {
+	display: block;
+}
+
+/* TODO: Add pagination css */
+.pagination-container {
+	margin: auto;
+}
+
+/* List Buttons To Render Items from Database */
+.resource-list-item {
+	display: flex;
+	flex-direction: column;
+	background-color: #2a2c2e;
+	margin-bottom: 20px;
+	padding: 36px;
+	border-color: transparent !important;
+	border-radius: 9px;
+	-webkit-box-shadow: 0px 10px 9px 0px rgba(0, 0, 0, 0.16);
+	-moz-box-shadow: 0px 10px 9px 0px rgba(0, 0, 0, 0.16);
+	box-shadow: 0px 10px 9px 0px rgba(0, 0, 0, 0.16);
+}
+
+.resource-list-item button {
+	padding-left: 0px;
+	margin-bottom: 10px;
+	border: 0;
+}
+
+.resource-list-title {
+	position: relative;
+	text-align: left;
+	padding: 0.1rem 0.75rem;
+	padding-left: 0px !important;
+	color: white;
+	border-bottom: 2px solid #ff9617 !important;
+	background-color: transparent !important;
+	font-size: min(4vw, 1.3rem);
+	font-family: serif;
+	transition: all 300ms;
+}
+
+.resource-list-title:hover {
+	color: #ff9617;
+	z-index: 1;
+	text-decoration: none;
+}
+
+.resource-list-title.active {
+	/* background-color: #535353; */
+	transition: all 300ms;
+}
+
+.resource-list-description {
+	list-style: none;
+	position: relative;
+	max-height: 0;
+	width: 100%;
+	opacity: 0;
+	padding-left: 0.75rem;
+	font-size: min(4vw, 1.2rem);
+	transition: all 300ms ease; /* controls fade out */
+	transform-origin: top center; /* Content expands downward from top */
+}
+
+.resource-list-description.active {
+	max-height: 50rem; /* Large number that the height should never reach */
+	opacity: 1;
+	padding: 0.5rem 0.75rem;
+	animation: growDown 300ms ease-in-out forwards;
+}
+
+.resource-list-description p {
+	color: white; /* description text color */
+	font-size: 16px;
+	margin: 10px 0px;
+}
+
+@keyframes growDown {
+	0% {
+		transform: scaleY(0);
+	}
+	80% {
+		transform: scaleY(1.1); /* The bounce makes it look snappy */
+	}
+	100% {
+		transform: scaleY(1);
+	}
+}
+
+/* Formats links */
+a.resource-link {
+	color: #ff9617;
+	padding: 0em;
+	font-size: 16px;
+	text-decoration: underline;
+}
+
+/* Tags */
+.tagsContainer {
+	display: flex;
+	flex-wrap: wrap;
+	margin-top: 10px;
+	z-index: 100;
+}
+
+.tag {
+	margin-right: 10px;
+	margin-top: 5px;
+	margin-bottom: 5px;
+	padding: 0px 5px;
+	font-size: 16px;
+	cursor: pointer;
+	color: white;
+	width: fit-content;
+	text-align: center;
+	border: 1.5px solid #2196f3;
+	border-radius: 2px;
```

Comment:
The border weight is thinner than the design but I think yours fit better making the view not too focused on tags. I think the border-radius should be increased to 5px. It looks like a sharp rectangle with 2px.